Engine/Component/ComponentAABB.cpp

```diff
@@ -129,12 +129,9 @@ bool ComponentAABB::IsEmpty() const
 	return bounding_box.Size().Length() == 0;
 }
 
-Component* ComponentAABB::Clone(bool original_prefab) const
+Component* ComponentAABB::Clone(GameObject* owner, bool /*original_prefab*/) 
```

Comment:
why bool parameter is commented?

Reply:
is not use, some IDEs has warning as error and not used parameter can be an error.